docs/sources/dashboards/create-reports/index.md

```diff
@@ -107,11 +107,15 @@ You can include dynamic dashboards with panels or rows, set to repeat by a varia
 
 By default, reports use the saved time range of the dashboard. You can change the time range of the report by:
 
-- Saving a modified time range to the dashboard.
+- Saving a modified time range to the dashboard. Changing the dashboard time range without saving it doesn't change the time zone of the report.
 - Setting a time range via the **Time range** field in the report form. If specified, the custom time range overrides the time range from the report's dashboard.
 
 The page header of the report displays the time range for the dashboard's data queries. Dashboards set to use the browser's time zone use the time zone on the Grafana server.
 
+#### Report time zones
+
+To avoid confusing time zone behavior in your reports, we recommend against using the **Browser Time** time zone setting in the dashboard from which you're generating a report. Instead, set a specific time zone for the dashboard. That way reports always show the correct time zone regardless of what your default time zone is when you create a report.
+
```

Comment:
```suggestion
The page header of the report displays the time range for the dashboard’s data queries.

#### Report time zones

Reports use the time zone of the dashboard from which they’re generated. Dashboards with the **Browser Time** setting use the time zone of the Grafana server and this applies to the dashboard's reports as well. This time zone may not match the time zone of users creating or receiving the report.

You can control the time zone for your reports by setting the dashboard to a specific time zone. Note that this affects the display of the dashboard for all users. 

```

Reply:
I like it. 
```
Dashboards with the **Browser Time** setting use the time zone of the Grafana server and this applies to the dashboard's reports as well
```
I prefer something like: 
```
Dashboards with the **Browser Time** setting use the time zone of the client browser. There is no Client browser on report generation, in this case reports will use the Grafana server time zone.
```

Reply:
This phrase is confusing to me:
`Dashboards with the **Browser Time** setting use the time zone of the Grafana server`
Who uses the time zone of the Grafana server? The dashboard or the report?

Reply:
I made a slight edit to that sentence but it was there before. But maybe that sentence isn't totally right either? I'm not sure about adding in the step of the client server though; I think it moves the conclusion too far away when we can just skip that and say Browser time = Grafana server time for reports like so:

`Reports use the time zone of the dashboard from which they’re generated. 

If a dashboard has the **Browser Time** setting, the reports generated from that dashboard use the time zone of the Grafana server. That's because there is no client browser associated with the report generation. As a result, this time zone might not match the time zone of users creating or receiving the report.`

Reply:
Actually I really don't like the client browser sentence in there. I think it just adds a layer of thinking that doesn't need to be there. As long as users know that for hte report Browser time = Grafana server time, I think it gets the job done.

```
Reports use the time zone of the dashboard from which they’re generated. 

If a dashboard has the **Browser Time** setting, 
the reports generated from that dashboard use the time zone of the Grafana server. 
As a result, this time zone might not match the time zone of users creating or receiving the report.
```

Reply:
Sorry to bring more complexity here but reports actually use the time zone of the server on which the [Image Renderer](https://grafana.com/grafana/plugins/grafana-image-renderer/) is installed. It's the Grafana server when the image renderer is installed as a plugin but it can be another server when it's run as a remote service.

Reply:
For the sake of complexity, can we disregard this particular case 😅? Or maybe use `server` instead of `Grafana server`?

Reply:
@AgnesToulet with the text on line 119 (below) and if we use the word "server" instead of "Grafana server", does that resolve this issue?

Reply:
I didn't see the paragraph below. Then I think it makes sense to keep `Grafana server` here and, if it's different from the image renderer one, then users will read the paragraph below where we are saying that if it's weird, it's expected.

Reply:
```suggestion
The page header of the report displays the time range for the dashboard's data queries.

#### Report time zones

Reports use the time zone of the dashboard from which they’re generated. 

If a dashboard has the **Browser Time** setting, the reports generated from that dashboard use the time zone of the Grafana server. As a result, this time zone might not match the time zone of users creating or receiving the report.

```